commit 7b9cbf97e0e5686e92063661cf167f128e7c4b01
Author:     Dongkyun Son <dongkyun.s@samsung.com>
AuthorDate: Fri Oct 16 11:22:56 2026 +0900
Commit:     Dongkyun Son <dongkyun.s@samsung.com>
CommitDate: Fri Oct 16 12:26:12 2026 +0900

    Point Tizen-Base-X snapshots to Tizen-Base-Dev
    
    Temporary workaround for llvm build time regression caused by
    -fno-inline-functions + clang on x86_64. Tizen-Base-Dev has the
    fix (cb67996861d0) and builds normally.
    
    Co-Authored-By: Cline SR
    Change-Id: Ibfa06be5972ce3a87227b7f64a1655e5f6d069b1
---
 TIZEN/Tizen/Tizen-Base-X/snapshot_info | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/TIZEN/Tizen/Tizen-Base-X/snapshot_info b/TIZEN/Tizen/Tizen-Base-X/snapshot_info
index a9bf147..28d7e98 100644
--- a/TIZEN/Tizen/Tizen-Base-X/snapshot_info
+++ b/TIZEN/Tizen/Tizen-Base-X/snapshot_info
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <snapshotinfo>
-  <latest target="tizen-base-x_20261006.014753" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-X/"/>
-  <reference target="tizen-base-x_20261006.014753" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-X/"/>
-  <base target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/">
+  <latest target="tizen-base-dev_20261013.023909" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/"/>
+  <reference target="tizen-base-dev_20261013.023909" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/"/>
+  <base target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/">
     <repository name="standard">
       <type name="standard"/>
     </repository>
